src/Caching/IBulkReadStorage.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (https://nette.org)
+ * Copyright (c) 2004 David Grudl (https://davidgrudl.com)
+ */
+
+namespace Nette\Caching;
+
+
+/**
+ * Cache storage with a bulk read support.
+ */
+interface IBulkReadStorage extends IStorage
+{
+
+	/**
+	 * Read from cache.
+	 * @param  string key
+	 * @return array key => value pairs, missing items are omitted
+	 */
+	function bulkRead(array $keys);
```

Comment:
TODO: consider returning `[$value1, $value2]` instead of more complex `[$key1 => $value1, $key2 => $value2]`